ssh_check/datadog_checks/ssh_check/ssh_check.py

```diff
@@ -1,13 +1,27 @@
 # (C) Datadog, Inc. 2018
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
+import re
 import time
 from collections import namedtuple
 
 import paramiko
 
 from datadog_checks.checks import AgentCheck
 
+# Example ssh remote version: http://supervisord.org/changes.html
+#   - SSH-2.0-OpenSSH_8.1
+SSH_REMOTE_VERSION_PATTERN = re.compile(
```

Comment:
Doesn't match for me `OpenSSH_for_Windows_7.7p1, LibreSSL 2.6.5`:

![Capture](https://user-images.githubusercontent.com/9677399/71053834-ccaed700-211d-11ea-9e1c-48d7cb3915df.PNG)

Reply:
Added explicit test